gulpfile.js

```diff
@@ -26,17 +26,14 @@
 
 'use strict';
 
-var $ = require('gulp-load-plugins')({
```

Comment:
What was the significance of this change?

Reply:
`gulp-jsdoc-to-markdown` isn't supported in Gulp 4, I followed this recommendation: https://github.com/jsdoc2md/jsdoc-to-markdown/wiki/How-to-use-with-gulp

Reply:
Ah, I got you.  Thanks!